src/main/resources/application.properties

```diff
@@ -1,26 +1,28 @@
 server.port=8080
 
-#env=
+#env=dev
 #sentry.dsn=
 sentry.logging.enabled=true
 sentry.logging.minimum-event-level=warn
 
 aws.region=eu-west-3
-#aws.eventBridge.bus=<env>-hei-admin
-#aws.sqs.queueUrl=<env>-hei-admin
-#aws.cognito.userPool.id=<userPoolId>
+#aws.eventBridge.bus=
+#aws.sqs.queueUrl=
+#aws.cognito.userPool.id=${pool_id}
 aws.cognito.jwt.jwsAlgorithm=RS256
 aws.cognito.jwt.readTimeout=2000
 aws.cognito.jwt.connectTimeout=2000
-#test.aws.cognito.idToken=<id_token>
+#test.aws.cognito.idToken=
+
 
 # use the same json mapper as openapi-generator
 spring.mvc.converters.preferred-json-mapper=jackson
 
-#spring.datasource.url=jdbc:postgresql:hei_admin
+#spring.datasource.url=jdbc:postgresql:/hei_admin
 #spring.datasource.password=postgres
 spring.datasource.username=postgres
 spring.flyway.locations=classpath:/db/migration
 spring.jpa.hibernate.ddl-auto=none
 spring.jpa.show-sql=false
 spring.jpa.open-in-view=true
+spring.jpa.properties.hibernate.jdbc.time_zone=UTC+3
```

Comment:
If you set time zone as UTC+3 in database, note that Java Instant is not configured through this.